src/app/desktop/components/Instances/Instance.js

```diff
@@ -336,6 +340,20 @@ const Instance = ({ instanceName }) => {
           />
           Open Folder
         </MenuItem>
+
+        {/* // TODO - Support other export options besides curseforge forge. */}
```

Comment:
We could actually add a dropdown menu inside the export modal to choose the format to export to